cineast-core/build.gradle

```diff
@@ -150,6 +151,12 @@ dependencies {
     /** JavaCPP. */
     api group: "org.bytedeco", name: "javacpp", version: version_javacpp
     api group: "org.bytedeco.javacpp-presets", name: "ffmpeg", version: version_ffmpeg
+    
+    /** OpenCV. */
+    api group: 'org.openpnp', name: 'opencv', version: '4.5.1-2'
+
+    /** JTS */
+    api group: 'org.locationtech.jts', name: 'jts-core', version: '1.18.1'
```

Comment:
For the sake of consistency, it would make sense to externalize the versions into the gradle.properties file as was done for the remaining dependencies.